bangc-ops/mlu_op.h

```diff
@@ -2550,6 +2550,156 @@ mluOpSqrtBackward(mluOpHandle_t handle,
                   const mluOpTensorDescriptor_t dx_desc,
                   void *diff_x);
 
+// Group:Voxelization
+/*!
+ * @brief Gets extra space size that is needed in voxelization operation.
+ *
+ * @param[in] handle
+ * Handle to an MLUOP context that is used to manage MLU devices
+ * and queues in the voxelization operation.
+ * @param[in] points_desc
+ * The descriptor of the tensors. For detailed information, see
+ * ::mluOpTensorDescriptor_t.
+ * @param[in] voxel_size_desc
+ * The descriptor of the tensors. For detailed information, see
+ * ::mluOpTensorDescriptor_t.
+ * @param[in] coors_range_desc
+ * The descriptor of the tensors. For detailed information, see
+ * ::mluOpTensorDescriptor_t.
+ * @param[in] max_points
+ * An integer value which is the maximum points contained in a voxel.
+ * @param[in] max_voxels
+ * An integer value which is the maximum voxels this function create.
```

Comment:
An integer value which is the maximum number of voxels that this function creates.